control/tests/passivity_test.py

```diff
@@ -54,6 +114,26 @@ def test_ispassive_edge_cases(test_input, expected):
     assert(passivity.ispassive(sys) == expected)
 
 
+def test_ispassive_all_zeros():
+    A = numpy.array([[0]])
+    B = numpy.array([[0]])
+    C = numpy.array([[0]])
+    D = numpy.array([[0]])
+    sys = ss(A, B, C, D)
+
+    with pytest.raises(ValueError):
+        passivity.ispassive(sys)
+
+def test_rho_and_nu_are_none():
```

Comment:
Is this really a forbidden case? I see it is in the current implementation, but is it from the definition? Also, what's the difference between 0 and `None` exactly?

Reply:
I'm trying to use `None` as a marker to treat the index as a variable to be maximized. Trying to maximize both the input and output passivity indices is **bilinear** matrix inequality, versus a **linear** matrix inequality. The former is **not** convex.

Reply:
Alright, you make a distinction between the parameters for `ispassive()` and `solve_passivity_LMI()`. Got it.